### PR TITLE
MNTOR-1150: Fix email template header style

### PR DESCRIPTION
**Pull request by flozia**
# References: 
Jira: [MNTOR-1150](https://mozilla-hub.atlassian.net/browse/MNTOR-1150)

# Description

Only some of the email gods might know why, but for some reason, the logo header style got stripped by all email clients I checked. This also leads to the rest of the header having a layout shift in Outlook. This PR fixes both of those issues.

# Screenshot (if applicable)

image

# Checklist (Definition of Done)
- [x] Localization strings (if needed) have been added.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [x] I've added or updated the relevant sections in readme and/or code comments
- [x] I've added a unit test to test for potential regressions of this bug.
- [x] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [x] All acceptance criteria are met.
- [x] Jira ticket has been updated (if needed) to match changes made during the development process.
- [x] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

File: src/views/email-2022.js [hunk 1]
```diff
@@ -32,14 +32,7 @@ const tableStyle = `
   width: 100%;
 `
 
-const headeLogoStyle = `
-  background-size: 240px 50px;
-  background: #f9f9fa url('${images.logoLight}')
-  height: 100px;
-  no-repeat 50%;
-`
```
Comment (flozia) [inline src/views/email-2022.js:40]: This style is broken and should be:
```
const headeLogoStyle = `
  background-size: 240px 50px;
  background: #f9f9fa url('${images.logoLight}') no-repeat 50%;
  height: 100px;
`
```

But even then the inline style got stripped and the issue remained.

**Review by toufali — APPROVED** [review]
Interesting finds!  When I wrote the legacy code I tested in Outlook and I believe QA did too – not sure if something got lost in refactor translation?  In any event, thanks for being thorough and glad it works again!  Let's make sure QA tests some of the same edge cases you did 👍

**Comment by flozia** [comment]
> When I wrote the legacy code I tested in Outlook and I believe QA did too – not sure if something got lost in refactor translation?

Yes, I saw that working with V1 as well and tried to find any discrepancies — no luck. Let’s see if this does not surface any new issues otherwise I’ll dive back into it.
